### PR TITLE
TestCase added for Container Create with HostName

### PR DESCRIPTION
**Pull request by rajdeepd**
Signed-off-by: Rajdeep Dua dua_rajdeep@yahoo.com

### CODE REVIEW

**Comment by jfrazelle** [comment]
is this closing an issue? 

LGTM regardless, just want to be sure we clean house after :) thanks!

File: integration-cli/docker_api_containers_test.go [hunk 1]
```diff
@@ -681,6 +681,49 @@ func (s *DockerSuite) TestContainerApiCreate(c *check.C) {
...
+	}
+	var container createResp
+	if err := json.Unmarshal(b, &container); err != nil {
+		c.Fatal(err)
+	}
+
+	var id = container.Id
+	configGet := map[string]interface{}{}
+
+	_, bodyGet, err := sockRequest("GET", "/containers/"+id+"/json", configGet)
```
Comment (duglin) [inline integration-cli/docker_api_containers_test.go]: I think configGet can be replaced with "nil" and then you can remove line 705

**Comment by duglin** [comment]
Just one minor comment/nit.
It looks ok but I do wonder why you picked on "HostName" instead of testing all of the various options?  Is "HostName" special in some way?

**Comment by rajdeepd** [comment]
I will pick others as well, Shall i add it to separate test cases for each param.

**Comment by rajdeepd** [comment]
this is just a beginning, please to add cases for each param in the next few pull requests

**Comment by duglin** [comment]
@rajdeepd its up to you whether to add them to this or a new PR.  Just let us know if you plan to add more to this one and we'll wait to merge.

**Comment by rajdeepd** [comment]
Can you close this i will add more in the next PR..

**Comment by duglin** [comment]
Closing per author's request.

**Comment by rajdeepd** [comment]
i meant merge this and will create  a new one

**Comment by duglin** [comment]
oh I'm sorry about that - I see you opened a new one.

**Pull request closed without merging**